lib/models/hostapp.ts

```diff
@@ -33,7 +33,7 @@ export interface OsVersion {
 	osType: string;
 	line?: OsLines;
 	variant?: string;
-
+	archived?: boolean;
```

Comment:
It's not the version that is archived, but the entire hostapp, so it doesn't make sense to put it on the `Release`, so to speak.

Reply:
got your point @sradevski, I'm closing this PR to do another solutions